Header header

# request
MissionRequest request

# response to request (True, False)
bool response

# mission succesfully completed (True, False)
bool completed
